cdap-ui/app/cdap/styles/main.scss

```diff
@@ -16,12 +16,12 @@
 
 body {
   .cdap-container {
-    height: 100%;
+    height: inherit;
     .container-fluid {
-      height: 100%;
+      height: inherit;
       padding: 0;
       > div {
-        height: 100%;
+        height: inherit;
```

Comment:
will this work if we simply remove the height properties?

Reply:
Don't think so. `inherit` is just inheriting parent's height. If we remove that it defaults to `auto` which takes the height of its children.